programming/typescript/challenges/index.js

```
@@ -0,0 +1,22 @@
+* Demonstrates different ways to use the greet functionality
+*/
+var typescript_1 = require("@lehungio/typescript");
+var greetMessage = (0, typescript_1.greet)('Typescript'); // Output: "Hello, World!"
+console.log(greetMessage);
+var helloWorld = new Challenges.HelloWorld();
+helloWorld.greet(); // Output: "Hello, World!"
+/**
+* Challenge 25: Word Count
+* Demonstrates the word counting functionality
+*/
+var typescript_2 = require("@lehungio/typescript");
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)